model.py

```diff
@@ -31,13 +31,25 @@ class Shortlist:
 
 #functions
 
+
+def load_pickle(path):
+    with open(path/"shortlist.pickle", "rb") as f:
+        shortlist = pickle.load(f)
+    return shortlist
+
 # gets the data of related properties and creates a object shortlist consisting of the role and all the applicants
 def load_shortlist(path):
-    criteria = load_criteria(path+"/criteria.csv")
-    role = load_role(path,criteria)
-    applicants = load_applicants(path) 
-    shortlist = Shortlist(role,applicants)
-    return shortlist
+    file = path/"shortlist.pickle" 
+    if file.exists():
+        shortlist = load_pickle(path)
+        return shortlist
```

Comment:
You can move `return shortlist` to the end of this function (outside the condition) and remove from other condition too. Doesn't matter which condition creates shortlist variable, it's returned at end of function.